### ISSUE TITLE
I would like to hear the contents of the line when the screen reader announces a symbol in the symbol reference list

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#109523]
Issue Type: <b>Feature Request</b>

cc: @isidorn

By pressing shift + f12 on a symbol, the list of references for that symbol is read by the screen reader. The content announced is the name of the file in which the symbol is referenced, the line and the column. It seems to me that it would be interesting to hear the contents of the line.
also.

VS Code version: Code - Insiders 1.51.0-insider (e5c0007ce0e9a3816679cba674d03376c03726e3, 2020-10-27T05:33:58.966Z)
OS version: Linux x64 5.9.1-arch1-1

**Comment by isidorn** [comment]
I think this makes perfect sense, we do the same when debugging is paused - [code pointer](https://github.com/microsoft/vscode/blob/0b1cd268f1a2753ba02ab1a0754f7d4a0528b7b3/src/vs/workbench/contrib/debug/browser/debugService.ts#L804). We shuold append at the end of the aria label the content of the line. it should be at the end so users can skip over it in case it is the most verbose.
@jrieken assigning to you, and I can also tackle if you prefer that - in that case just unassign yourself.

Assigning for next milestone, since I think it should be a simple change

**Comment by jrieken** [comment]
I have pushed a change that adds the context around the match. This isn't the full line (since matches can occur very late in a line) but should be enough to understand where a match is. @jvesouza please gives it a try, will be in next insiders

**Comment by jrieken** [comment]
\closedWith ebaf0a2

**Comment by jvesouza** [comment]
I tested it and it was excellent. It will certainly improve my productivity.
Great job, thank you very much.
